src/Microsoft.Diagnostics.ExtensionCommands/NativeAddressHelper.cs

```diff
@@ -71,7 +71,7 @@ internal IEnumerable<DescribedRegion> EnumerateAddressSpace(bool tagClrMemoryRan
                     ClrRuntime clrRuntime = runtime.Services.GetService<ClrRuntime>();
                     if (clrRuntime is not null)
                     {
-                        foreach (ClrMemoryPointer mem in ClrMemoryPointer.EnumerateClrMemoryAddresses(clrRuntime))
+                        foreach (var mem in EnumerateClrMemoryAddresses(clrRuntime))
```

Comment:
Minor style nic: the coding convention that I follow (and I think the rest of the runtime team does) is to use `var` only if the type is obvious like with `new` of a class.  To me it makes the code more readable and like more "type" safe but I'm ok if you don't change these vars.

Reply:
I follow the same convention: I never use var except in an obvious type.  I'm not sure how these slipped through...

Please do point out style nits.  The style of the SOS codebase is very similar to how I prefer it anyway.